packages/v-analyzer/package.yaml

```
@@ -0,0 +1,32 @@
+---
+name: v-analyzer
+description: V language server.
+homepage: https://github.com/v-analyzer/v-analyzer
+licenses:
+  - MIT
+languages:
+  - V
+categories:
+  - LSP
+
+source:
+  id: pkg:github/v-analyzer/v-analyzer@latest
```

Using the `latest` release is problematic because it's an immutable release version that currently breaks the entire version management of the package. They seem to have provided an initial release, let's use that?
```suggestion
  id: pkg:github/v-analyzer/v-analyzer@v0.0.1-beta.1
```

---

Thanks for the review @williamboman . I see so this is the literal tag, not what is available via https://github.com/v-analyzer/v-analyzer/releases/latest? 

If we do so, is it possible to update the version short term when there is a new release? I.e. very recently we just added optimisation for the linux release binary, this change isn't included in `@v0.0.1-beta.1`. Or would nightly be okay since it's a young project? Nightly releases are provided: https://github.com/v-analyzer/v-analyzer/releases/tag/latest

---

> If we do so, is it possible to update the version short term when there is a new release? I.e. very recently we just added optimisation for the linux release binary, this change isn't included in `@v0.0.1-beta.1`.

Whenever there's a new release made Renovate will automatically pick it up and upgrade the version. While using the nightly version (`latest`) will technically be possible it will effectively render the version management of the package useless so it's not something we should do. If using the nightly build is the recommended way it might not be mature enough to be packaged in this registry yet.

If there are new changes that you'd like to get out it might make sense to cut a new release.